root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+  <div class="bubble" data-bind="bubble: $root.scriptBubble">
     <p>
-      [% l('The language attribute should be set to the language used for the release title and track titles. It should not be set to the language the lyrics are written in, nor to the language used for other extra information on the cover.') %]
+      [% l('The {doc|script} attribute should be set to the script used for the release title and track titles.',
+           { doc => { href => doc_link('Release#Script'), target => '_blank' } }) %]
+    </p>
+    <p>
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Release#Language_and_script'), target => '_blank' } }) %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.scriptBubble">
+  <div class="bubble" data-bind="bubble: $root.dateBubble">
+    <!-- ko ifnot: target() && (target().hasAmazonDate() || target().hasJanuaryFirstDate()) -->
+      <p>
+        [% l('The {doc|date} describes when this particular release was made available.',
+             { doc => { href => doc_link('Release/Date'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Release#Date'), target => '_blank' } }) %]
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && target().hasAmazonDate() -->
+      <p>
+        [% l('“1990-10-25” is the bogus date that Amazon gives to all releases for which they don’t know the actual date.
+              <strong>Please use this date only if you’re certain this date is correct.</strong>') %]
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && target().hasJanuaryFirstDate() -->
+      <p>
+        [% l('If you do not know the month or day of release, please leave them empty.
+              January 1st is not often the actual release date.
+              <strong>Please double-check that you have entered the release date correctly.</strong>') %]
+      </p>
+    <!-- /ko -->
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.countryBubble">
+    <p>
+      [% l('The {doc|country} describes where the release was made available.',
+           { doc => { href => doc_link('Release/Country'), target => '_blank' } }) %]
+    </p>
     <p>
-      [% l('The script attribute should be set to the script used for the release title and track titles.') %]
+      If you aren’t sure where it was released, please leave this unset.
     </p>
   </div>
 
+  <div class="bubble" data-bind="bubble: $root.catalogNumberBubble">
+    <!-- ko ifnot: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+      <p>
+        [% l('The {doc|catalog number} is assigned by the label and can often be found on the spine or near the barcode on physical media.',
+             { doc => { href => doc_link('Release#Catalogue_number'), target => '_blank' } }) %]
```

Comment:
The "catalogue" (rather than "catalog") in the documentation should probably be changed per https://github.com/metabrainz/design-system/blob/master/guidelines/style-guidelines.md#spelling, but I assume that doing that would break a bunch of existing links using fragments.

Reply:
We don't seem to have any links to that officially, but it might break some on edit notes. That's likely to happen anyway if we migrate the docs to ReadTheDocs or the like though so it might not be a big deal?

Reply:
Ah, I didn't even notice that there's a better page using the "correct" spelling: https://musicbrainz.org/doc/Release/Catalog_Number

I've edited https://wiki.musicbrainz.org/Release to use "catalog" and link to the Release/Catalog_Number page, if you don't mind reviewing/transcluding it.